### PR TITLE
Make workflows work fine in forked repos

### PR DESCRIPTION
**Pull request by mathbunnyru**
## Checklist (especially for first-time contributors)

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: .github/workflows/sphinx.yml [hunk 1]
```diff
@@ -25,7 +25,6 @@ on:
 jobs:
   build-docs:
     name: Build Sphinx Documentation
-    if: github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository == 'jupyter/docker-stacks')
```
Comment (mathbunnyru) [inline .github/workflows/sphinx.yml:28]: This should run just fine even in forked repos.

**Comment by mathbunnyru** [comment]
I guess a few more things have to be done to make it work really well:
- disable aarch64 builds
- be more careful with wiki updates

I will deal with these issues a bit later.

**Comment by bjornjorgensen** [comment]
https://github.com/bjornjorgensen/docker-stacks/pull/8

**Comment by mathbunnyru** [comment]
@bjornjorgensen I finally made it work properly. Please, merge the latest `main` to your active branches if you need to.

**Pull request merged**